docs/system-admin-guide/authentication/oauth-applications/README.md

```diff
@@ -8,34 +8,35 @@ keywords: OAuth application settings
 ---
 # OAuth applications
 
-To activate and configure oauth applications, navigate to -> *Administration* -> *Authentication* and choose -> *Oauth applications*.
+To activate and configure OAuth applications, navigate to -> *Administration* -> *Authentication* -> *OAuth applications*.
 
-## Add a new authentication application for oauth
+## Add a new authentication application for OAuth
 
-To add a new oauth application, click the green **+ Add** button.
+To add a new OAuth application, click the green **+ Add** button.
 
-![Sys-admin-authenticatoin-oauth-applications](Sys-admin-authenticatoin-oauth-applications.png)
+![Sys-admin-authenticatoin-OAuth-applications](Sys-admin-authenticatoin-OAuth-applications.png)
```

Comment:
Careful when rigorously search and replace "oauth" with "OAuth". The file path here is wrong now.